Sources/Rendering/Core/ImageMapper/test/testSlicingModePosition.js

```diff
@@ -0,0 +1,42 @@
+import test from 'tape-catch';
+
+import vtkImageMapper from 'vtk.js/Sources/Rendering/Core/ImageMapper';
+import { SlicingMode } from 'vtk.js/Sources/Rendering/Core/ImageMapper/Constants';
+import vtkImageSlice from 'vtk.js/Sources/Rendering/Core/ImageSlice';
+import vtkRTAnalyticSource from 'vtk.js/Sources/Filters/Sources/RTAnalyticSource';
+
+test('Test slice position differences between XYZ and IJK modes', (t) => {
+  const source = vtkRTAnalyticSource.newInstance({
+    wholeExtent: [0, 10, 0, 10, 0, 10],
+  });
```

Comment:
Test with a spacing != 1 and an orientation that's not the identity if you want to test differences between XYZ and IJK.